rs/replicated_state/src/canister_state/system_state.rs

```diff
@@ -1326,6 +1326,9 @@ impl SystemState {
     /// another subnet.
     pub fn drop_in_progress_management_calls_after_split(&mut self) {
         // Remove aborted install code task.
+        //
+        // Note that this cannot be a paused install code task, because we abort all
+        // paused tasks before triggering the split.
```

Comment:
Isn't this function also used in the legacy subnet split where calls are only aborted afterwards? If so I'm not sure I understand this comment here.